### ISSUE TITLE
Default behavior doesn't seem to work

### ISSUE THREAD

**Issue by venantius** [greglook/whidbey#17]
I'm getting around to https://github.com/venantius/ultra/issues/25, but the `0.6.0` release here doesn't seem to work when used in isolation. At the moment using `0.5.1` with Whidbey active but Ultra off, I get the following: 

![screen shot 2015-03-14 at 6 44 02 pm](https://cloud.githubusercontent.com/assets/1824859/6654337/300d5cea-ca7a-11e4-80e3-76091279f71a.png)

But if I upgrade to `0.6.0` the colorization goes away entirely. Seems that the initialization isn't hooking in properly somewhere? 

![screen shot 2015-03-14 at 6 44 07 pm](https://cloud.githubusercontent.com/assets/1824859/6654339/361fa19c-ca7a-11e4-8759-76aa57f88aa7.png)

**Comment by greglook** [comment]
Hmm, are you running the repl in a leiningen project, or standalone?

**Comment by venantius** [comment]
When I filed this issue I was just doing it as a standalone, but it doesn't seem to work in a leiningen project either.

I created a new project with the following project.clj:

```clojure
(defproject whidbey-test "0.1.0-SNAPSHOT"
  :description "FIXME: write description"
  :url "http://example.com/FIXME"
  :license {:name "Eclipse Public License"
            :url "http://www.eclipse.org/legal/epl-v10.html"}
  :dependencies [[org.clojure/clojure "1.6.0"]]
  :plugins [[mvxcvi/whidbey "0.6.0"]]
  )
```

And then I emptied out my `~/.lein/profiles.clj`

Running `lein repl` gives me just the normal boring REPL :(

**Comment by venantius** [comment]
Add'l debug info: Clojure 1.6.0, Java 1.8.0_11, Leiningen version 2.5.1, Yosemite 10.10.2

**Comment by greglook** [comment]
I've been swamped lately at work, but I'll try to investigate this later this week.

**Comment by venantius** [comment]
No worries, I've got plenty to keep me busy :)

**Comment by greglook** [comment]
Interesting, I upgraded Leiningen to 2.5.1 and I see it fail in some projects:

```
REPL-y 0.3.5, nREPL 0.2.6
Clojure 1.6.0
OpenJDK 64-Bit Server VM 1.7.0_79-b14
```

But it succeeds on another project using Clojure 1.7:

```
REPL-y 0.3.5, nREPL 0.2.6
Clojure 1.7.0-beta1
OpenJDK 64-Bit Server VM 1.7.0_79-b14
```

**Comment by venantius** [comment]
Peculiar. Given that 1.6.0 and 2.5.1 are the current release versions for Clojure and Lein respectively it would be ideal to figure out a way to get this working with those. Odd bug, though.

**Comment by greglook** [comment]
I added some tracing to whidbey's initialization logic and it looks like in 2.5.1 there's no `:repl` profile for the `lein repl` invocation:

```
TRACE: (:base :system :user :provided :dev)
TRACE: false
TRACE: nil
TRACE: nil
TRACE: :else
TRACE: (:base :system :user :provided :dev)
TRACE: false
TRACE: nil
TRACE: nil
TRACE: :else
```

This works on the other project because I have an explicit `:repl` profile defined in the project file.

**Comment by greglook** [comment]
I'll dig into the Leiningen changes between 2.4.3 and 2.5.1, I believe I saw some change notes about repl profiles. It might be time to switch back to including the plugin only in the :repl profile and dropping the detection logic.

**Comment by greglook** [comment]
This should be fixed by b00eb1f1d6e894ba3a94918a50d6ec25b7d4d601, which is available in the 1.0.0 release! It turned out to be _much_ cleaner to use the new profile metadata features in Leiningen.
